# First and second order formation statistics

float64 m_x
float64 m_y

float64 m_xx
float64 m_xy
float64 m_yy